languages/elixir/exercises/concept/date-parser/.docs/introduction.md

```diff
@@ -15,15 +15,15 @@ The `=~/2` operator is useful to perform a regex match on a string to return a `
 # => true
 ```
 
-> Two notes about using sigils:
->
-> - many different delimiters may be used depending on your requirements rather than `/`
-> - string patterns are already _escaped_, when writing the pattern as a string not using a regex, you will have to _escape_ backslashes (`\`)
+Two notes about using sigils:
+
+- many different delimiters may be used depending on your requirements rather than `/`
+- string patterns are already _escaped_, when writing the pattern as a string not using a regex, you will have to _escape_ backslashes (`\`)
 
 Matching a range of characters using square brackets `[]` to denote a _character class_. This will match any one character to the characters in the class. You can also specify a range of characters like `a-z`, as long as the start and end represent a contiguous range of code points.
 
 ```elixir
-regex = ~r/[a-z][AZ][0-9][!?]/
+regex = ~r/[a-z][ADKZ][0-9][!?]/
```

Comment:
It took me a while to notice that there's no dash between A and Z, so I didn't understand this example right away.